gulpjs/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# Use the base App Engine Docker image, based on debian jessie.
+FROM debian:8
+
+# Install updates and dependencies
+RUN apt-get update -y && apt-get install --no-install-recommends -y -q curl python build-essential git ca-certificates libkrb5-dev imagemagick && \
+    apt-get clean && rm /var/lib/apt/lists/*_*
+
+# Install the latest LTS release of nodejs
+RUN mkdir /nodejs && curl https://nodejs.org/dist/v6.9.1/node-v6.9.1-linux-x64.tar.gz | tar xvzf - -C /nodejs --strip-components=1
+ENV PATH $PATH:/nodejs/bin
+RUN npm install -g gulp-cli
```

Comment:
We are going to discuss it internally first. I'll come back to you.
If you could provide a real example, that would be very helpful.